ludwig/datasets/download.py

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env python
+# coding=utf-8
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+from io import BytesIO
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+
+"""A mixin to simulate downloading the zip file containing the training data
+and extracting the contents"""
+
+
+class ZipDownloadMixin:
+
+    """Download the raw dataset and extract the contents
+    of the zip file and store that in the cache location.
+    args:
```

Comment:
No need to specify empty args and empty return value.

Reply:
Fixed